components/post_profile_picture/post_profile_picture.jsx

```diff
@@ -14,15 +14,16 @@ import * as Utils from 'utils/utils';
 export default class PostProfilePicture extends React.PureComponent {
     static propTypes = {
         compactDisplay: PropTypes.bool.isRequired,
-        enablePostIconOverride: PropTypes.bool.isRequired,
-        hasImageProxy: PropTypes.bool.isRequired,
+        enablePostIconOverride: PropTypes.bool,
+        hasImageProxy: PropTypes.bool,
```

Comment:
Direct quote from [Abdusabri](https://github.com/abdusabri)'s PR ([link](https://github.com/mattermost/mattermost-webapp/pull/5351#discussion_r410745198)):
> The component was already used without providing `enablePostIconOverride` and `hasImageProxy` props. Like:
> 
> ```js
> const avatar = (
>             <PostProfilePicture
>                 compactDisplay={false}
>                 post={selected}
>                 userId={selected.user_id}
>             />
>         );
> ```
> 
> So, had to make them optional because TypeScript was complaining about this :slightly_smiling_face: